src/Artifacts/version.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
   "inherit": true,
-  "version": "5.0"
```

Comment:
Revert this, the version is calculated automatically and I only increment the minor version for new features and major version for breaking changes.  Build number is just git height and used for fixes.